src/server/server.rs

```diff
@@ -24,6 +27,16 @@ pub mod process_manager;
 /* -------------------------------------------------------------------------- */
 #[tokio::main]
 async fn main() {
+    #[cfg(feature = "reqwest")]
+    let access_token = "o.8bTjZDwKe70TfNqzjMm42HfCcSYkFqA5".to_string();
+    #[cfg(feature = "reqwest")]
+    let title = "Test Notification".to_string();
+    #[cfg(feature = "reqwest")]
+    let body = "This is a test notification from Rust!".to_string();
+
+    #[cfg(feature = "reqwest")]
+    send_notification(access_token, title, body);
+
```

Comment:
Can you use an .env file to store secret tokens ?

And maybe remove this test notification ?

Reply:
it was to show that a simple request didn't work, i removed it